src/shared/proto/geometry.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+message Point
+{
+    // x coordinate
+    float x = 1;
+
+    // y coordinate
+    float y = 2;
```

Comment:
Why not make these doubles?

Reply:
fixed

Reply:
conversely, why make these doubles? Floats should have plenty of precision, and if we (maybe) want this to work with radio smaller messages are better. Same reason I pointed the same stuff out in @jonathanlew 's pr for robot status messages.

Reply:
doubles would make it consistent with `Point` in Software

Reply:
> doubles would make it consistent with Point in Software

I think it's more important to consider the context of what the message is used for / how its used. 

> But even with floats, Its next to impossible to fit all this stuff in 100 bytes.

That's fair but I don't think is a good reason to intentionally take up more space because it already doesn't fit. This would result in more fragments (should we implement that at some point) which could affect throughput.

This really doesn't matter until we fully commit to supporting old robots or not (I don't remember what our condition was. We talked a while back), and it's easy to change, so I'm fine with this for now.